mmv1/products/compute/ServiceAttachment.yaml

```diff
@@ -207,7 +207,7 @@ properties:
             create.
   - !ruby/object:Api::Type::Boolean
     name: reconcileConnections
-    default_value: true
+    default_from_api: true
```

Comment:
Why is this change needed? Is the default value changed for this field?

Reply:
> We want to default from the API instead of setting the default explicitly at terraform